src/main/java/org/tron/common/runtime/RuntimeImpl.java

```diff
@@ -412,8 +479,13 @@ private void call()
       logger.info("No contract or not a smart contract");
       throw new ContractValidateException("No contract or not a smart contract");
     }
-    byte[] code = this.deposit.getCode(contractAddress);
+
     long callValue = contract.getCallValue();
+    if (isNewVersion() && callValue < 0) {
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Change this condition so that it does not always evaluate to "false" ([squid:S2583](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2583))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1670&issues=AWbSdHUuSgN4uCclEWnE&open=AWbSdHUuSgN4uCclEWnE)

[ISSUE_KEY]: AWbSdHUuSgN4uCclEWnE